querytests/appuio_managed_vcpu.jsonnet

```diff
@@ -50,7 +50,10 @@ local baseSeries = {
     core: '0',
   }, '1x120'),
 
-  appuioInfoLabel: c.series('appuio_managed_info', infoLabels, '1x120'),
+  appuioInfoLabel: c.series('appuio_managed_info', infoLabels, '1x60 _x60'),
```

Comment:
Shouldn't this be

```suggestion
  appuioInfoLabel: c.series('appuio_managed_info', infoLabels, '1x30 _x90'),
```

Reply:
I forced a overlap between the two series since promtool has a different staleness handling (i think™️).